src/lib/index.ts

```diff
@@ -42,6 +42,9 @@ class ServerlessIamPerFunctionPlugin {
               defaultInherit: {
                 type: 'boolean',
               },
+              iamGlobalPermissionsBoundary: {
+                type: 'string',
```

Comment:
Should be clarified and replaced with [`'#/definitions/awsArnString'`](https://github.com/serverless/serverless/blob/v2.16.1/lib/plugins/aws/provider.js#L257)

Reply:
Do you mean that I should create a function that validate permissionsBoundary string and replace de type with '#/definitions/awsArnString'?

something like this:
```js
/**
     * @param {string} policyArn
     * @returns void
     */
    validatePermissionsBoundaryArn(policyArn) {
        if (lodash_1.default.isEmpty(policyArn)) {
            return;
        }

        const validation = /^arn:aws:iam::(\s*|\d{12}):policy\/.+/;
        if (!validation.test(policyArn)) {
            const errorMessage = [
                'iamPermissionsBoundary and iamGlobalPermissionsBoundary should be a Policy ARN',
            ].join('');
            this.throwError(errorMessage);
        }
    }
```

Reply:
The first is correct.
```
iamGlobalPermissionsBoundary: { $ref: '#/definitions/awsArnString' }
```

Reply:
updated.

Reply:
perfect